.gitignore

```diff
@@ -157,6 +157,9 @@ app/.vscode/
 
 app/routers/stam
 
+bin
+routes 1.py
```

Comment:
This shouldn't be in the .gitignore file. You should just not commit it.

Reply:
You're right it came from a `pull` :)